### PR TITLE
[FEATURE] Subtask - update README.md

### PR DESCRIPTION
**Pull request by MatthewK3023**
1. Add the Link of Decanter and Decanter SDK introduction page
2. List the example code(python script and jupyter) and explain what the diff and purpose
3. Add the link of brief development guide and flow
4. Add System requirement (python3.7)

### CODE REVIEW

**Review by NoctisHong — CHANGES_REQUESTED** [review]
Need to explain more detail between Jupyter and Python example.
Why does it need to sue asyncio?
Where's the loaction they placed?

**Review by schwannden — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -40,28 +45,35 @@ $ python -m example.file
 Progress UploadTask_train:  55%|█████████████████████████████████████████
 ```
 
-## Auto Time Series Example: Upload Data, Train Time Series Forecast Experiment and Predict
-train data path: `examples/data/tsf/iris_train.csv`  
-test data path: `examples/data/tsf/iris_train_test.csv`
+## Example Dataset Path 
+* `examples/data/data` - store the general structure dataset
```
Comment (NoctisHong) [inline README.md:49]: store the general dataset just under `examples/data/`

File: README.md [hunk 2]
```diff
@@ -40,28 +45,35 @@ $ python -m example.file
...
 
 ## Example Code
-* Python Script: [example.py](https://github.com/MoBagel/decanter-ai-core-sdk/blob/master/examples/example.py)
-* Jupyter: [jupyter_example.ipynb](https://github.com/MoBagel/decanter-ai-core-sdk/blob/master/examples/jupyter_example.ipynb)
-
-
-## Development Guide and Flow
-* If you are curious why Decanter AI Core SDK does certain things the way it does and not differently, visit our [Development Guide](https://mobagel.github.io/decanter-ai-core-sdk/notes/design.html)
-
+* General Structure Data
```
Comment (NoctisHong) [inline README.md:54]: Just let it be "General Data"

File: README.md [hunk 3]
```diff
@@ -41,6 +46,21 @@ $ python -m example.file
...
+Note: In Jupyter, it will initially exist a running event loop.
+```python
+import asyncio
+loop = asyncio.get_running_loop()
+loop.is_running()
+```
+
+
+## Development Guide and Flow
+* If you are curious why Decanter AI Core SDK does certain things the way it does and not differently, visit our [Development Guide](https://mobagel.github.io/decanter-ai-core-sdk/notes/design.html)
```
[the remark this answers is not in the document]
Comment (MatthewK3023) [reply README.md]: done

File: README.md [hunk 4]
```diff
@@ -57,6 +77,8 @@ For guidance on setting up a development environment and how to make a contribut
 ## Links
 For more details on design, guidance on setting up a development environment, and SDK usage.
 
+* Introduction about Decanter AI: https://mobagel.com/solution
```
[the remark this answers is not in the document]
Comment (MatthewK3023) [reply README.md]: done

File: README.md [hunk 5]
```diff
@@ -57,6 +77,8 @@ For guidance on setting up a development environment and how to make a contribut
...
+* Introduction about Decanter AI SDK: https://mobagel.github.io/decanter-ai-core-sdk/
```
[the remark this answers is not in the document]
Comment (MatthewK3023) [reply README.md]: done

File: README.md (hunk 1 above)
Comment (MatthewK3023) [reply README.md]: done

File: README.md (hunk 2 above)
Comment (MatthewK3023) [reply README.md]: done

**Review by NoctisHong — APPROVED** [review]

**Pull request merged**